bot/processors/matchmaking.py

```diff
@@ -54,47 +54,52 @@ def request_match_blocks(channel_id):
 def confirm_match_request(channel_id):
     blocks = [
         {
-            "type": "section",
-            "text": {
-                "type": "mrkdwn",
-                "text": f"Awesome, we will match you with other users in <#{channel_id}> soon!",
+            'type': 'section',
+            'text': {
+                'type': 'mrkdwn',
+                'text': f'Awesome, we will match you with other users in <#{channel_id}> soon!',
             }
         }
     ]
 
     return blocks
 
 
-def create_match_blocks(topic_channel_id, conversation_id):
+def create_match_blocks(topic_channel_id, conversation_id, reminder=False):
```

Comment:
Consider this change. I'm squishing together the two blocks so that it's easier to change all of the text at once. I'm also attempting to make the new message more "playful". See what you think.

```python
def create_match_blocks(topic_channel_id, conversation_id, reminder=False):
    if reminder:
        message = f'''Hey! Were you guys able to meet for the <#{topic_channel_id}> discussion?

If not, there's still time. Just click the button below when you're ready to set up the Penny Chat.'''
    else:
        message = f'''Yahoo, you\'ve been matched for a conversation about <#{topic_channel_id}>!

Work together to find a time to meet and chat. Once you do, click the button below to schedule a Penny Chat.'''

    blocks = [
        {
            'type': 'section',
            'text': {
                'type': 'mrkdwn',
                'text': message,
            }
        },
...
```

Reply:
Yeah, I like it.